pythonwhat/State.py

```diff
@@ -181,22 +181,14 @@ def has_different_processes(self):
             # play it safe (most common)
             return True
 
-    # todo: remove (deprecated)
-    def assert_root(self, fun, extra_msg=""):
-        if self.parent_state is not None:
-            raise InstructorError(
-                "`%s()` should only be called from the root state, `Ex()`. %s"
-                % (fun, extra_msg)
-            )
-
     def assert_execution_root(self, fun, extra_msg=""):
-        if self.parent_state is not None and not self.assert_creator_type("run"):
+        if not (self.is_root or self.is_creator_type("run")):
```

Comment:
This is blocked until `proto` is bumped to next version, correct?

Reply:
Yes